configs/default/EASP-RRIOTF722.config

```
@@ -0,0 +1,92 @@
+# Betaflight / STM32F7X2 (S7X2) 4.4.3 Nov 14, 2023 / 22:53:32 (738127e) MSP API: 1.45
+#mcu STM32F7X2
```

```suggestion
#mcu STM32F7X2

```
i wouldn't mind an empty-line here, but such is getting picky.